doc/akri_connector/connector-metadata-schema.json

```diff
@@ -0,0 +1,230 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "aioConnectorMetadataSchemaVersion": "1.0.0",
+  "definitions": {
+    "AdditionalConfiguration": {
+      "type": "object",
+      "description": "The schema for this 'additional configuration' field. Must provide inline schema xor the file that contains the schema",
+      "properties": {
+        "schema": {
+          "$ref": "http://json-schema.org/draft-07/schema#"
+        },
+        "schemaFile": {
+          "type": "string"
+        }
+      },
+      "oneOf": [
+        {
+          "required": [
+            "schema"
+          ]
+        },
+        {
+          "required": [
+            "schemaFile"
+          ]
+        }
+      ]
+    }
+  },
+  "properties": {
```

Comment:
@varunpuranik Do we need a section here for generic connector secret alias list?

Reply:
Yes, we do.. this should include teh list of secrets and additional config that the connector needs

Reply:
For ref `ConnectorTemplate` property: https://msazure.visualstudio.com/One/_git/akri?path=/deployment/helm/examples/akri.yaml&version=GBmain&line=561&lineEnd=562&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents